projects/ngrx.io/content/guide/signals/signal-store/entity-management.md

```diff
@@ -179,6 +155,48 @@ patchState(
 );
 ```
 
+### `setEntity`
+
+Adds or replaces an entity in the collection.
+
+```ts
+patchState(store, setEntity(todo));
+```
+
+### `setEntities`
+
+Adds or replaces multiple entities in the collection.
+
+```ts
+patchState(store, setEntities([todo1, todo2]));
+```
+
+### `setAllEntities`
+
+Replaces the current entity collection with the provided collection.
+
+```ts
+patchState(store, setAllEntities([todo1, todo2, todo3]));
+```
+
+### `upsertEntity`
+
+Adds or updates an entity in the collection.
+When updating, it does not replace the existing entity but merges it with the provided one.
```

Comment:
I would add a note about unspecified properties and how these are handled.
The reason is because there were some questions/discussions about it in @ngrx/entity.

An example can be found in the tests where the age property isn't set to a user, and thus the original age of the user remained the same.

Reply:
I've added "it does not replace the existing entity but merges it with the provided one" to describe that behavior. Do you have a suggestion on how to improve this paragraph for more clarity?

Reply:
Something like this?

```suggestion
When updating, it does not replace the existing entity but merges it with the provided one. Only the properties provided in the update are merged with the existing entity. Properties not present in the update are left unchanged.
```